spek-ide-plugin/intellij-common/src/main/kotlin/org/spekframework/intellij/SpekRunConfigurationProducer.kt

```diff
@@ -0,0 +1,31 @@
+package org.spekframework.intellij
+
+import com.intellij.execution.actions.ConfigurationContext
+import com.intellij.execution.actions.RunConfigurationProducer
+import com.intellij.execution.configurations.ConfigurationTypeUtil
+import com.intellij.openapi.util.Ref
+import com.intellij.psi.PsiElement
+
+class SpekRunConfigurationProducer: RunConfigurationProducer<SpekBaseRunConfiguration<*>>(
+    ConfigurationTypeUtil.findConfigurationType(SpekBaseConfigurationType::class.java)
+) {
+    override fun isConfigurationFromContext(configuration: SpekBaseRunConfiguration<*>,
+                                            context: ConfigurationContext): Boolean {
```

Comment:
Just a personal thing — I think this can be better:

```kotlin
override fun isConfigurationFromContext(
    configuration: SpekBaseRunConfiguration<*>,
    context: ConfigurationContext
): Boolean {
```

Reply:
ack